core/src/epicli/data/common/ansible/playbooks/roles/helm/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Get list of Helm repos 
+  become: yes
+  shell: |-
+    set -o pipefail &&
+    helm repo list | grep {{ helm_chart_repo_name }}
+  args:
+    executable: /bin/bash
+  register: helm_repo_list
+  failed_when: "helm_repo_list.rc == 1"
+
+- name: Check if files exist in {{ specification.apache_epirepo_path }}/helm-charts
+  find:
+    paths: "{{ specification.apache_epirepo_path }}/helm-charts/system"
+  register: helm_charts_files_number
+
+- name: "Add Helm chart repo and install charts"
+  block:
+    - name: Get Helm charts list from repo before update 
+      become: yes
+      shell: helm search repo {{ helm_chart_repo_name }} --output json | jq -r '.[].name'
+      register: helm_charts_list_before_update
+
+    - name: "Add {{ helm_chart_repo_name }} Helm repository from url {{ repository_url }}"
+      become: yes
+      shell: helm repo add {{ helm_chart_repo_name }} {{ repository_url }}/helm-charts/system
+      when: 
+        - helm_chart_repo_name not in helm_repo_list.stdout 
+
+    - name: Update Helm repo
+      become: yes
+      shell: helm repo update
+
+    - name: Get Helm charts list from repo after update 
+      become: yes
+      shell: helm search repo {{ helm_chart_repo_name }} --output json | jq -r '.[].name'
+      register: helm_charts_list_after_update
+
+    - name: Delete not existing Helm charts 
+      become: yes
+      shell: |- 
+        chart_subname=$(echo {{ item }} | cut -d "/" -f 2)
+        helm delete $chart_subname
+      loop: "{{ helm_charts_list_before_update.stdout_lines }}"
+      when: item not in helm_charts_list_after_update.stdout_lines
+
+    - name: Install Helm charts 
+      become: yes
+      shell: |- 
+        chart_subname=$(echo {{ item }} | cut -d "/" -f 2)
```

Comment:
Please use the upper case variable name here. I'd suggest using built-in `jinja2/ansible` filters instead of `cut`.

Reply:
If you're looking for a `split` filter please check here https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#creating-valid-variable-names :+1: